cmake/project_version.cmake

```diff
@@ -0,0 +1,39 @@
+# Sets GIT_EXECUTABLE
+find_package(Git REQUIRED)
+
+if (NOT ${Git_FOUND})
+  message(FATAL_ERROR "Git not found.")
+endif()
+
+# We could remove --tags if we started annotating the release tags.
+execute_process(
+  COMMAND ${GIT_EXECUTABLE} describe --tags HEAD
+  RESULT_VARIABLE git_error
+  OUTPUT_VARIABLE git_describe
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+if (git_error)
+  message(FATAL_ERROR "Failed to run git describe: ${git_error}")
```

Comment:
This means you can no longer build the downloaded archive.

Reply:
> This means you can no longer build the downloaded archive.

Should be fixed now.